intro-HPC/ch_running_jobs_with_input_output_data.tex

```diff
@@ -307,6 +307,66 @@ \subsection{Your scratch space (\$VSC\_SCRATCH)\label{subsec:scratch-directory}}
 of writing there are no plans to provide this.
 \end{description}
 
+\ifgent
+\subsection{Your UGent home drive and shares \label{subsec:ugent-shares}}
```

Comment:
my main issue here is that this is part of "running jobs" instructions, and this is only relevant for the loginnodes, so not running jobs.

i would add some text here that "in order to access data on your UGent share(s), you need to stage-in the data (and stage-out afterwards)" or something like that. otherwise, this section is in wron location.

@kwaegema we can configure the compute nodes too with the shares config, so people can run inetractive jobs straight on the shares. (need to change the routing to nat02 on all nodes for that too, but that is ok)
but it's not really possible to access shares from within regular jobs, and users should not really try to do so as this will imply security risks

Reply:
@stdweird I know it's not a very good place to put it, but I don't know where else :) Well, I first thought about 3.3
Transfer Files to/from the HPC, that is now only about scp/sftp and stuff, but maybe that's a good place after all..